music/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.urls import reverse
+
+class Album(models.Model):
+    artist = models.CharField(max_length=250)
+    album_title = models.CharField(max_length=500)
+    genre=models.CharField(max_length=100)
+    album_logo=models.CharField(max_length=1000)
+
+    def get_absolute_url(self):
+        reverse('music:detail',kwargs={'pk':self.pk})
+
+
+    def __str__(self):
+        return self.album_title +' , '+ self.artist
+
+class Song(models.Model) :
+    album=models.ForeignKey(Album,on_delete=models.CASCADE)
+    file_type=models.CharField(max_length=10)
+    song_title=models.CharField(max_length=250)
+    is_favorite=models.BooleanField(default=False)

```

Comment:
Boolean field are by default False so no need to define the default argument unless to set it to `true`